src/Store/InputStoreConnector.re

```diff
@@ -46,7 +46,10 @@ let start = (window: option(Revery.Window.t), runEffects) => {
   let handleTextEffect = (state: State.t, k: string) => {
     switch (Model.FocusManager.current(state)) {
     | Editor
-    | Wildmenu => [Actions.KeyboardInput(k)]
+    | Wildmenu => [
+        Actions.KeyboardInput(k),
+        Actions.Hover(Feature_Hover.KeyPressed(k)),
```

Comment:
Just curious - why do we have to add this specifically for the wildmenu?

Reply:
Oh that's interesting. I guess my formatting was bad and it lumped it in there when I ran `esy format`. You're right that it shouldn't be 👍

Reply:
Ah it's not specifically for the wild menu. I just added it in where the editor case was. I can split those if you think it makes sense.

Reply:
Ah, I see, I missed that the `Editor` was in the same handler